src/main/java/uk/gov/justice/digital/delius/controller/InstitutionalReportController.java

```diff
@@ -0,0 +1,118 @@
+package uk.gov.justice.digital.delius.controller;
```

Comment:
General comment, I thought we were going to surface specific types of institutional report, eg parole report?